Recipes/Recipes.Public/Tag.cs

```diff
@@ -2,6 +2,6 @@
 
 public class Tag
 {
-    public int Id { get; set; }
+    public long Id { get; set; }
```

Comment:
Just curious to why we needed to make this a long as opposed to int as we did elsewhere?

Reply:
Sorry, force of habit :D will change